### ISSUE TITLE
Example doesn't  work

### ISSUE THREAD

**Issue by ghost** [aaronsw/html2text#2]
Example on home page doesn't  work

http://www.aaronsw.com/2002/html2text/?url=http://daringfireball.net/

**Comment by aaronsw** [comment]
This looks like a result of the change to html.parser in #1. I guess it should be filed as an html.parser bug. (It remains in 3.1, sadly.)

**Comment by aaronsw** [comment]
Looks like this is http://bugs.python.org/issue670664

Not sure what to do about it.

**Comment by ghost** [comment]
Yes :(. That bug in HTMLParser live 8 years - topic started 2003. A month age - last comment, so there is a chance to be fixed )
